src/components/EligibilityForm/components/ContactForm.tsx

```diff
@@ -10,15 +11,21 @@ import ContactInformation, {
   defaultValuesContactInformation,
   validationSchemasContactInformation,
 } from './ContactInformation';
-import { ContactFormFooter } from './EligibilityForm.styled';
+import {
+  ContactFormFooter,
+  FormFreezer,
+  FormWrapper,
+} from './EligibilityForm.styled';
 
 type ContactFormProps = {
   onSubmit: (values: FormikValues) => void;
-  isSubmitting?: boolean; // TODO: remove and replace by 'disable'
+  isLoaded?: boolean;
+  isDisabled?: boolean;
 };
 export const ContactForm = ({
   onSubmit,
-  isSubmitting = false,
+  isLoaded,
+  isDisabled,
```

Comment:
why do we need 2 differents props ? If something is loading it's automatically disabled ?

Reply:
No, indeed, it is no longer necessary.